node/silkworm/rpc/kv_calls.hpp

```diff
@@ -147,6 +151,7 @@ class TxCall : public BidirectionalStreamingRpc<remote::KV::AsyncService, remote
     static mdbx::env* chaindata_env_;
     static boost::posix_time::milliseconds max_ttl_duration_;
     static uint32_t next_cursor_id_;
+    static uint32_t max_cursor_id_;
```

Comment:
Does the cursor-id must be unique per connection or across all connections?

Reply:
Good point. I've checked this on Erigon again: cursor ID is unique and restarts from 0 (zero) for each new Tx call, so the check on ID wrapping is not needed because there is the max cursors per txn check in place.

I'm going to amend the PR. Thanks for spotting this.